OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -28,10 +28,10 @@ public class AttackFollowInfo : AttackBaseInfo
 
 	public class AttackFollow : AttackBase, INotifyOwnerChanged
 	{
-		protected Target requestedTarget;
+		public Target RequestedTarget;
```

Comment:
This is only made public because `HeliAttack` and `FlyAttack` are not part of the trait itself. After the upcoming Attack* refactor, I doubt if it still would need to be public at all.